src/data/hr.prepare.json

```diff
@@ -21,7 +21,7 @@
       "read_config": {
         "file_filter": "mks-dump/kvkadr.sql.gz",
         "container": "handelsregister",
-        "filter_list": "GRANT|DROP|OWNER|search_path|REVOKE|CREATE TRIGGER|CREATE INDEX|ADD CONSTRAINT|ALTER TABLE|PRIMARY KEY \\(",
+        "filter_list": "^GRANT|^DROP|OWNER|search_path|REVOKE|CREATE TRIGGER|CREATE INDEX|ADD CONSTRAINT|ALTER TABLE|^SET default_table_access_method|PRIMARY KEY \\(",
```

Comment:
De `filter_list` is nu gelijk aan de oude HR repo.

- `^DROP` voorkomt dat “DROPSTUFF MEDIA” en “HOLLANDDROP” uit de data verdwijnen
- `^GRANT` voorkomt dat “GRANT KLUSBEDRIJF”

Dat is mooi, maar als ik naar de dumps kijk (met name `kvkves.sql`) zie ik nog meer (potentiële) problemen die met nauwkeurigere reguliere expressies voorkomen kunnen worden.

- `OWNER` komt ook voor in de data bijv. in `kvkves.sql`: “THE NEXT GENERATION CO-OWNERSHIP SOFTWARE”
De regel die uitgefilterd moet worden is `ALTER TABLE igp_pgplup_cmg_owner.kvkvesm00 OWNER TO igp_pgplup_cmg_owner;`. Maar `ALTER TABLE` staat er al in, dus kan `OWNER` niet gewoon weg?
- `search_path` is prima maar zelf zou ik voor `^SELECT pg_catalog\.set_config` gaan. Nauwkeuriger en vangt ook andere eventuele config wijzigingen.
- `REVOKE` zie ik nergens terug in de data. Ik zou er in ieder geval `^REVOKE` van maken, analoog aan `^GRANT`.
- `CREATE TRIGGER` en `CREATE INDEX` zou ik ook een anchor geven; je kunt er dan `^CREATE (INDEX|TRIGGER)` van maken.
- Iets als `ADD CONSTRAINT` is nodig omdat de `ALTER TABLE ONLY igp_pgplup_cmg_owner.kvkvesm00
    ADD CONSTRAINT kvkvesml0 PRIMARY KEY (vesid);` regel over twee regels is verdeeld. Dus prima.
- `ALTER TABLE` staat ook altijd aan het begin van de regel en verdient ook een anchor.
- `^SET default_table_access_method` is prima!
- `PRIMARY KEY \\(` wordt al afgevangen door de `ADD CONSTRAINT` en is dus niet nodig.

Reply:
Tnx. verwerkt!